cmake/Modules/FindWebp_ep.cmake

```diff
@@ -0,0 +1,130 @@
+# FindWebp_EP.cmake
+#
+# The MIT License
+#
+# Copyright (c) 2022 TileDB, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+# Fetches and builds libwebp for use by tiledb.
+# TBD...
+# This module defines:
+#   - libmagic_INCLUDE_DIR, directory containing headers
+#   - libmagic_LIBRARIES, the Magic library path
+#   - libmagic_FOUND, whether Magic has been found
+#   - The libmagic imported target
+
+# Include some common helper functions.
+include(TileDBCommon)
+
+# Search the path set during the superbuild for the EP.
+set(WEBP_PATHS ${TILEDB_EP_INSTALL_PREFIX})
+
+if (TILEDB_WEBP_EP_BUILT)
+  find_package(webp PATHS ${TILEDB_EP_INSTALL_PREFIX} ${TILEDB_DEPS_NO_DEFAULT_PATH})
+endif()
+
+if (TILEDB_WEBP_EP_BUILT)
+  find_path(webp_INCLUDE_DIR
+    NAMES encode.h decode.h demux.h mux.h mux_types.h
+    PATHS ${WEBP_PATHS}
+    PATH_SUFFIXES include/webp
+    ${NO_DEFAULT_PATH}
+  )
+
+  # Link statically if installed with the EP.
+  # TBD: Does this find/include the multiple libraries comprising webp?
+  find_library(webp_LIBRARIES

```

Comment:
See above, switch to find_package and libraries are provided as targets.

Reply:
changed to use WebP targets.